docker-compose.yml

```diff
@@ -28,6 +28,24 @@ services:
     ports:
       - ${PORT__HEALTHCHECK_METRICS:-7300}:7300
 
+  eigenda-proxy:
+    platform: linux/amd64
+    image: us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/eigenda-proxy:v1.4.1
+    restart: unless-stopped
+    stop_grace_period: 5m
+    entrypoint: /scripts/start-eigenda-proxy.sh
+    volumes:
+      - eigenda-data:/data
+      - ./scripts/:/scripts
+    ports:
+      - ${PORT_EIGENDA_PROXY:-4242}:4242
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    deploy:
+      # If USE_LOCAL_EIGENDA_PROXY is unset or empty, then we set replicas to 1,
+      # otherwise use the value of USE_LOCAL_EIGENDA_PROXY which should be 0.
+      replicas: ${USE_LOCAL_EIGENDA_PROXY:-1}
```

Comment:
Creative use of the two string interpolation modifiers! Not easy to read in combination, but it works well enough with the comments.